ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -42,28 +42,28 @@
 
     <template  v-if="isMhrManufacturerRegistration">
       <!-- Attention -->
-      <section id="mhr-attention" class="mt-13">
+      <section id="mhr-attention" class="mt-15">
         <Attention
-          section-id="mhr-attention"
-          :section-number="1"
+          sectionId="mhr-attention"
+          :sectionNumber="1"
           :validate="isValidatingApp"
           @isAttentionValid="setAttentionValidation"
         />
       </section>
 
       <!-- Folio or Reference Number -->
-      <section id="mhr-folio-or-reference-number" class="mt-13">
+      <section id="mhr-folio-or-reference-number" class="mt-15">
         <FolioOrReferenceNumber
-          section-id="mhr-folio-or-reference-number"
-          :section-number="2"
+          sectionId="mhr-folio-or-reference-number"
+          :sectionNumber="2"
```

Comment:
Perhaps the dynamic prop binding is due to satisfying the prop type of `number`, just keep in mind, if it was a literal prop type vue would handle the type conversion. Although vue would pass it as a string before converting to a number. 

Food for thought.

Reply:
To clarify, no action required here :)